pkg/asset/installconfig/ovirt/validation.go

```diff
@@ -2,9 +2,9 @@ package ovirt
 
 import (
 	"fmt"
-
```

Comment:
This blank line still needs to be put back.

Reply:
Done